whitepaper/whitepaper.md

```diff
@@ -1187,10 +1188,31 @@ Web 3.0 applications (decentralized apps or *DApps*) which interact with blockch
 
 <div style="page-break-after: always; visibility: hidden"><!-- \pagebreak --></div>
 
-# 7. References
+# 7. Related Work
+Blockchain interoperability is emerging as one of the crucial features of blockchain technology
+A recent survey classifies blockchain interoperability studies in three categories: Cryptocurrency-directed interoperability approaches, Blockchain Engines, and Blockchain Connectors [[4](#8-references)]. Each category is further divided into sub-categories based on defined criteria. 
+Each category serves particular use cases. 
+
+![rw](./related-work-categories.png)
+
+Cryptocurrency-directed interoperability approaches identify and define different strategies for chain interoperability across public blockchains, most of them implementing cryptocurrencies.
+
+Blockchain engines are frameworks that provide reusable data, network, consensus, incentive, and contract layers for the creation of customized blockchains, serving general use-cases. Emerging blockchains are, for example, the Cosmos Network and Polkadot. 
+
+The Blockchain Connector category is composed of interoperability solutions that are not cryptocurrency-directed or blockchain engines. Several sub-categories exist: Trusted Relays, Blockchain Agnostic Protocols, Blockchain of Blockchains, and Blockchain Migrators”.
+
+White Hyperledger Cactus has caracteristics from the the three categories, it can be considered a Blockchain Connector (namely a Trusted Relay). In particular, Cactus focuses on providing multiple use case scenarios via a trusted consortium. 
+Trusted relays allow the discovery of the target blockchains, appearing often in a permissioned blockchain environment, where cross-blockchain transactions are routed by trusted escrow parties.
+ 
+ However, depending on the validator plugin, the trust on the relay can be decentralized, making Cactus a decentralized, general-purpose, trustless relay. 
+It is possible that Cactus becomes a Blockchain of blockchains, given that it will provide mechanisms for developers to build cross-chain dApps. Conversely, the blockchain migrator feature paves the way for building a solution that performs data migration across blockchains.
```

Comment:
@RafaelAPB 
`It is possible that Cactus becomes a Blockchain of blockchains`

Never say never for sure, but I think we've stated the opposite a few times so this may confuse the reader. 
Also depends on how the phrase `blockchain of blockchains` is understood, from an abstract, theoretical perspective you could call it a blockchain of blockchains if you just meant by it that one has access to multiple blockchains through a Cactus deployment, but the way we used the term so far is when `blockchain of blockchains` means that there's yet another blockchain storing actual data (blocks) and it's somehow "meta" relative to the rest of the chains.
Long story short, at the moment I'd say we'd rather not provide a full blown implementation of a distributed ledger (blockchain) but just aim to connect existing software that does that.

Also possible that I'll end up looking like a fool in the future for saying all this, but I'm at peace with that. :-)

Reply:
Yes, agreed! I meant this because, in fact, a blockchain of blockchains might just correspond to a fancy validator plugin, tracking the decentralized voting process.